demo-forms/custom_contacts/README.MD

```diff
@@ -0,0 +1,11 @@
+# Applying Sample Custom Contact Types
+
+
+1. Download the current `app_settings.json`
+1. Replace the existing `contact_types` with the contents of [contact_types.json](contact_types.json)
+1. Upload the change to `app_settings.json`
```

Comment:
```suggestion
1. Upload the change to `app_settings.json` by using `medic-conf`.  This should work if you're in the `cht-core/config/default` directory: `medic-conf --local compile-app-settings upload-app-settings`
```

Reply:
I added the details using the admin app.